src/Components/PastMenu.js

```diff
@@ -0,0 +1,254 @@
+      <div className="row justify-content-center">
+        <h1>{this.props.menu}</h1>
+        <p className="col-lg-10 col-12">
+          {this.pastMenuDict[this.props.menu].description}
+        </p>
+        <div className="align-center background">
+          <h1 className="section-header">Starters</h1>
+          <p className="col-lg-10 col-12">
+            The starters for this menu, the starters are served on a shared
+            plate for all to indulge in.
+          </p>
+          <Carousel interval={null} className="col-lg-8 col-12">
+            {this.pastMenuDict[this.props.menu].starters.map((starter) => (
+              <Carousel.Item key={starter.title}>
+                <div
+                  className="past-menu car-item"
+                  style={{ backgroundImage: `url(${starter.image})` }}
+                ></div>
+                <Carousel.Caption>
+                  <div className="past-menu-caption">
+                    <h3>{starter.title}</h3>
+                    <p>{starter.description}</p>
+                  </div>
+                </Carousel.Caption>
+              </Carousel.Item>
+            ))}
+          </Carousel>
+        </div>
+        <div className="align-center">
+          <h1 className="section-header">Mains</h1>
+          <p className="col-lg-10 col-12">
+            For large groups (8-12pax) we will have additional main served as
+            indicated on the main with "(8-12pax)" in the title.
+          </p>
+          <Carousel interval={null} className="col-lg-8 col-12">
+            {this.pastMenuDict[this.props.menu].mains.map((main) => (
+              <Carousel.Item key={main.title}>
+                <div
+                  className="past-menu car-item"
+                  style={{ backgroundImage: `url(${main.image})` }}
+                ></div>
+                <Carousel.Caption>
+                  <div className="past-menu-caption">
+                    <h3>{main.title}</h3>
+                    <p>{main.description}</p>
+                  </div>
+                </Carousel.Caption>
+              </Carousel.Item>
+            ))}
+          </Carousel>
+        </div>
+        <div className="align-center background">
+          <h1 className="section-header">Dessert</h1>
+          <p className="col-lg-10 col-12">
+            There is only one dessert choice to round of the meal.
+          </p>
+          <div
+            className="past-menu car-item dessert-container col-lg-8 col-12"
+            style={{
+              backgroundImage: `url(${
+                this.pastMenuDict[this.props.menu].desserts[0].image
+              })`,
+            }}
+          >
+            <div className="past-menu-caption dessert-caption">
+              <h3>{this.pastMenuDict[this.props.menu].desserts[0].title}</h3>
+              <p>
+                {this.pastMenuDict[this.props.menu].desserts[0].description}
+              </p>
+            </div>
+          </div>
+        </div>
```

Comment:
Would be nice to create separate components for this, as it would make more sense when reading it, as to what it is.